docker-compose.yml

```diff
@@ -6,7 +6,6 @@ services:
     - postgres
     - redis
     - redis-cache
-    - netbox-worker
```

Comment:
Why did you remove the worker here?

Reply:
The worker is not needed before the Webserver is started. The worker can also be started after the Webserver.